php/site-utils.php

```diff
@@ -143,27 +143,55 @@ function setup_site_network( $site_name ) {
 
 }
 
+/**
+ * Reloads configuration of ee-nginx-proxy container
+ *
+ * @return bool
+ */
+function reload_proxy_configuration() {
+	return EE::exec( 'docker exec ee-nginx-proxy sh -c "/app/docker-entrypoint.sh /usr/local/bin/docker-gen /app/nginx.tmpl /etc/nginx/conf.d/default.conf; /usr/sbin/nginx -s reload"' );
+}
+
 /**
  * Adds www to non-www redirection to site
  *
- * @param string $site_name Name of the site.
- * @param bool $le          Specifying if letsencrypt is enabled or not.
+ * @param string $site_name name of the site.
+ * @param bool $ssl         enable ssl or not.
+ * @param bool $inherit     inherit cert or not.
  */
-function add_site_redirects( $site_name, $le ) {
+function add_site_redirects( string $site_name, $ssl, $inherit ) {
```

Comment:
Please maintain code consistency. here you're defining data type of function variable and other places you're not. Either you define it for all or not at all.
ref: https://github.com/EasyEngine/easyengine/pull/1171/files#diff-43605c4fabcca6b901ccbab377546cfeR404

Reply:
Resolved.